cmd/dc4bc_cli/main.go

```diff
@@ -260,6 +268,30 @@ func getSignaturesCommand() *cobra.Command {
 					fmt.Println()
 				}
 			}
+
+			if jsonOutputFolder == "" {
+				return nil
+			}
+
+			f, err := os.OpenFile(jsonOutputFolder, os.O_WRONLY|os.O_CREATE, 0600)
```

Comment:
why do you open it if it's a folder? 🤔 
btw, why is it a folder at all? according to the task, it should be a json file with an array of signatures